app/org/sagebionetworks/bridge/config/BridgeSpringConfig.java

```diff
@@ -475,6 +482,35 @@ public DynamoDBMapper activityDdbMapper(DynamoUtils dynamoUtils) {
         return dynamoUtils.getMapper(DynamoScheduledActivity.class);
     }
 
+    @Bean
+    public SessionFactory hibernateSessionFactory() {
+        // Hibernate configs
+        Properties props = new Properties();
+        props.put("hibernate.connection.driver_class", "com.mysql.jdbc.Driver");
+        props.put("hibernate.connection.requireSSL", true);
+        props.put("hibernate.dialect", "org.hibernate.dialect.MySQLDialect");
+
+        // c3p0 connection pool properties
+        props.put("hibernate.c3p0.min_size", 5);
+        props.put("hibernate.c3p0.max_size", 20);
+        props.put("hibernate.c3p0.timeout", 300);
+        props.put("hibernate.c3p0.idle_test_period", 300);
+
+        // Connection properties come from Bridge configs
+        BridgeConfig config = bridgeConfig();
+        props.put("hibernate.connection.password", config.get("hibernate.connection.password"));
+        props.put("hibernate.connection.url", config.get("hibernate.connection.url"));
+        props.put("hibernate.connection.username", config.get("hibernate.connection.username"));
```

Comment:
They actually don't. As long as the configs are defined somewhere, they don't need to be stubbed out in the conf we ship with Github.